ansible/roles/dokuwiki/files/sudoers.d/dokuwiki

```diff
@@ -0,0 +1,2 @@
+Defaults!/opt/dokuwiki/libexec/update_user env_keep=PASSWORD
```

Comment:
This willl not scale for all app/hook combinations. Let's put the`invoke hook` script in sudoers, exactly how you said in back channel.

Besides that, your `sudoers.d` scheme is not going to work properly, you can only have one line per user. So if you'd have `dokuwiki` and `hoover` both under `sudoers.d`, both setting a command for user `liquid`, only the last imported one is going to be used.